poc_v2.1_revised/openai_llm.py

```
@@ -0,0 +1,39 @@
+            response = self.llm.invoke(prompt)  # Use .invoke() for ChatOpenAI
+            return response.content  # Access the content of the ChatMessage
+        except Exception as e:
+            print(f"Error calling OpenAI: {e}")
+            return "Sorry, there was an error processing your request with OpenAI."
+        
+    def standardize_query(self, question: str) -> str:
+        prompt = self.staging_prompt_template.format_messages(question=question)
+        try:
+            response = self.llm.invoke(prompt)  # Use .invoke() for ChatOpenAI
+            return response.content  # Access the content of the ChatMessage
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)